packages/cli/src/commands/join.ts

```diff
@@ -141,12 +141,23 @@ export async function handleJoin(argv: JoinOptions, config: Config, packageVersi
     }
   }
 
+  const seenVersions = new Set();
```

Comment:
```suggestion
  let seenVersions;
```

Reply:
I simplified the code a bit so that Set() isn't required.